serializers/src/glTF/2.0/glTFExporter.ts

```diff
@@ -1259,6 +1259,11 @@ export class _Exporter {
                     glTFNodeIndex = this._nodeMap[babylonNode.uniqueId];
                     if (glTFNodeIndex !== undefined) {
                         glTFNode = this._nodes[glTFNodeIndex];
+
+                        if (babylonNode.metadata) {
+                            glTFNode.extras = babylonNode.metadata;

```

Comment:
As I noted [here](https://github.com/BabylonJS/Babylon.js/pull/6505#discussion_r295907422), it might be a good idea to only export specific parts of the metadata. I think ideally we expose a callback so that users can decide what metadata to export and default to exporting `metadata.gltf.extras`.

Reply:
how would you like to expose it?

Reply:
fixed!